dtmi/wurthelektronik/desigkkit/calypsoiotkit-1.json

```diff
@@ -0,0 +1,193 @@
+[{
+		"@id": "dtmi:wurthelektronik:designkit:calypsoiotkit;1",
+		"@type": "Interface",
+		"displayName": "Calypso IoT Design Kit",
+		"description": "Reports temperature, relative humidity, pressure, acceleration and voltage and sets LED.",
+		"@context": [
```

Comment:
nit: move the context to the top of the file

Reply:
Done.